initialize/main.go

```diff
@@ -64,7 +64,7 @@ func readProjectConfig(command string) {
 	viper.SetDefault("Tokaido.Stability", "edge")
 	viper.SetDefault("Tokaido.Dependencychecks", true)
 	viper.SetDefault("Tokaido.Enableemoji", emojiDefaults())
-	viper.SetDefault("Tokaido.Phpversion", "7.1")
+	viper.SetDefault("Tokaido.Phpversion", "7.2")
```

Comment:
Ah cool, good call. Might pay to set calendar notifications to update this value on latest/LTS release. Maybe worth a wider discussion on how we handle defaults for this and other software versions

https://www.php.net/supported-versions.php

Got a pretty regular schedule for PHP now, looks like 7.2 will move out of LTS in December, replaced with 7.3

Reply:
Yeah good idea - I'll invite you to a calendar item to review in 3 months. 

We should also incorporate this page: https://www.drupal.org/docs/8/system-requirements/php-requirements in that equation